app/controllers/application_controller.rb

```diff
@@ -87,7 +87,7 @@ def skip_sentry_exception_types
       case exception
       when Pundit::NotAuthorizedError
         Common::Exceptions::Forbidden.new(detail: 'User does not have access to the requested resource')
-      when JWT::ExpiredSignature
+      when OpenidApplicationController::ExpiredTokenError
```

Comment:
Just a note: @kfrz We may run into a merge here in our ApplicationController Error Handling refactor

Reply:
If there is anything I can do to make that merge conflict go smoother, please let me know! I would happy to change my code to conform to yours.